core/src/main/java/com/scalar/db/sql/DdlStatementExecutor.java

```diff
@@ -23,9 +26,11 @@
 public class DdlStatementExecutor implements DdlStatementVisitor<Void, Void> {
```

Comment:
The cache invalidations are kicked when a schema change happens in `DdlStatementExecutor`.